.github/workflows/release.yml

```diff
@@ -0,0 +1,101 @@
+# This automation performs release tagging and artifact uploads of the LFS objects
+# Contact SIG-Build for any issues or looking to make changes
+
+name: Publish new release
+
+on:
+  # Allows you to run this workflow manually from the Actions tag
+  workflow_dispatch:
+    inputs:
+      repository:
+        description: 'The repo to target for release'
+        required: true
+        type: choice
+        default: o3de
+        options:
+          - o3de
+          - o3de-atom-sampleviewer-changml
```

Comment:
Is this the right repo?

Reply:
Yup, leftover from a test I did in my own fork. Fixed